denis/configure.py

```diff
@@ -97,16 +100,18 @@ def remove(assignment):
         print(f'no such assignment {assignment}')
 
 
-def dump():
-    def timestamp_to_isoformat(timestamp):
-        return datetime.fromtimestamp(timestamp).astimezone().isoformat()
+def dump(fmt_iso):
+    def timestamp_to_formatted(timestamp):
+        fmt = '%A, %B %d, %Y %I:%M:%S %p %Z (%z)'
```

Comment:
since this is only used once, maybe it could just go directly in the argument for strftime. I also think that you could use %a instead of %A and %b instead of %b  to get consistent length. %r is exactly equivalent to `%I:%M:%S %p` but much shorter, though I would personally perfer %T for 24 hour notation (`%H:%M:%S`).